interactive_engine/executor/ir/graph_proxy/src/adapters/gs_store/read_graph.rs

```diff
@@ -652,13 +737,13 @@ where
 /// and Some means we need given properties (and Some(vec![]) means we do not need any property)
 /// while in ir, None means we do not need any properties,
 /// and Some means we need given properties (and Some(vec![]) means we need all properties)
+/// now, storage use the same semantic as ir
+#[cfg(feature = "columns_filter_pushdown")]
```

Comment:
if column filter is not push down, the ir layer assume that it could get all props from and v/e item, so there is no need to call `encode_storage_prop_keys`;

Reply:
I know that only columns whin` param.columns` will be used in futher compute, so I use `encode_storage_prop_keys` for `to_runtime_vertex`